demo/react/components/notification/default.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { Fragment, useContext } from 'react';
+
+import { Button, NotificationTheme } from 'LumX';
+import {
+    NotificationProvider,
+    NotificationState,
+    notificationContext,
+} from 'LumX/components/notification/react/NotificationProvider';
+
+/////////////////////////////
+
+interface IProps {
+    /**
+     * The theme to use to display this demo.
+     */
+    theme: NotificationTheme;
+}
+
+/////////////////////////////
+
+/**
+ * The demo for the default <Notification>s.
+ *
+ * @return {React.ReactElement} The demo component.
+ */
+const DemoComponent: React.FC<IProps> = ({ theme }: IProps): React.ReactElement => (
+    <Fragment>
+        <NotificationProvider>
+            <NotificationClient theme={theme} />
+        </NotificationProvider>
+    </Fragment>
+);
+
+const NotificationClient: React.FC<IProps> = ({ theme }: IProps): React.ReactElement => {
+    const { close, error, info, success, warning }: NotificationState = useContext(notificationContext);
```

Comment:
why don't you use a custom hook like `useNotification()` ?

Reply:
What would be the purpose here?

Reply:
The idea behind creating a custom would be to hide all the context stuff (that could change in the future) so any component could just have to add a `useNotification()` to get acces to the notification system